zcash_client_sqlite/src/lib.rs

```diff
@@ -900,6 +925,36 @@ mod tests {
             .unwrap();
     }
 
+    #[cfg(feature = "transparent-inputs")]
+    #[test]
+    fn transparent_receivers() {
+        use secrecy::Secret;
+        use tempfile::NamedTempFile;
+        use zcash_client_backend::data_api::WalletReadTransparent;
+
+        use crate::{chain::init::init_cache_database, wallet::init::init_wallet_db};
+
+        let cache_file = NamedTempFile::new().unwrap();
+        let db_cache = BlockDb::for_path(cache_file.path()).unwrap();
+        init_cache_database(&db_cache).unwrap();
+
+        let data_file = NamedTempFile::new().unwrap();
+        let mut db_data = WalletDb::for_path(data_file.path(), network()).unwrap();
+        init_wallet_db(&mut db_data, Some(Secret::new(vec![]))).unwrap();
+
+        // Add an account to the wallet.
+        let (ufvk, taddr) = init_test_accounts_table_ufvk(&db_data);
+        let taddr = taddr.unwrap();
+
+        let receivers = db_data.get_transparent_receivers(0.into()).unwrap();
```

Comment:
It would be nice to run this test with two different seeds; one that triggers the case where `receivers` contains two different addresses, and one that triggers the case where there is only one.